models.py

```diff
@@ -3,7 +3,7 @@
 from sqlite3 import Row
 from typing import Optional
 
-from usermanager import User, UserFilters
+from lnbits.extensions.usermanager import UserFilters
```

Comment:
- direct imports from other extensions are not allowed now
- the depended on extension (`usermanager`) can be updated. Direct python imports cannot be updated.
- for this particular case just copy the `UserFilters` class

Reply:
artifact of https://github.com/lnbits/lnbits/pull/1545